docs/api-testing/mark3/mocking.md

```diff
@@ -0,0 +1,271 @@
+---
+id: mocking
+title: API Mocking with Piestry
+sidebar_label: Mocking
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+What is Sauce Labs _Piestry_? A server that feeds solely on OpenAPI 3 specs. Provides dynamicity in the responses so that proper positive, negative and edge case testing can be performed. Effortless mocks. No extra work, just works.
+
+Why use mocking?
+* Allows you to create stubbed API is that you can use to in your testing flow.
+* You can get rid of any dependencies from third-party APIs that may be expensive, but you don’t have control over it.
+* Allows you to also get ahead of your testing when your API is still in development and not built out yet.
+* You could re-create it in our mocking platform, write all of your tests against it, and then you can have this jumpstart on testing before your API is out of development.
+
+Why the name Piestry?
+didn't want to drop the Sauce tradition of naming stuff after food, so Piestry is:
+"A pastry that acts as if it was a pie, but is nothing more than a pastry"
+
+
+--use cases: you want to fake the payment transaction, you want to isolate the microservice from the rest of the API actions (ie everything else is stable and you want to drill down to find the error)
+
+
+Where can I run it?
+* Docker container (locally or CI/CI pipeline)
+  * Image: quay.io/saucelabs/piestry
+  * Run it using the code snippet below:
+  ```bash
+  docker run -v "$(pwd)/specs:/specs" -p 5000:5000 quay.io/saucelabs/piestry -u /specs/myspec.yaml
+  Where /specs/myspec.yaml is the URI to a YAML file, local or remote.
```

Comment:
where $(pwd)/myspec means the {current_directory}/myspec that gets mounted to the container in the /specs folder.
Therefore, the -u (relative to the container is) /specs/myspec.yaml